C++/include/data_structure/stack/stack.hpp

```diff
@@ -0,0 +1,81 @@
+/*
+    Stack
+    ------------------
+    A data structure which has the Last In First Out (LIFO) property. Elements are added
+    and removed from the top(head). Only the top element can be accessed at any given time.
+    Hence, there is no random access. All member functions have O(1) time complexity.
+*/
+#ifndef stack_h
+#define stack_h
+
+#include "data_structure/linked_list/singly_linked_list.hpp"
+#include <iostream>
+
+template<class T>
+class Stack {
+private:
+    int size;
```

Comment:
I think it would be better to use `size_t` to represent size. Return this from `length()`, too.